specification/containerservice/Fleet.Management/tspconfig.yaml

```diff
@@ -2,7 +2,8 @@ emit:
   - '@azure-tools/typespec-autorest'
 options:
   '@azure-tools/typespec-autorest':
-    azure-resource-provider-folder: '{cwd}/../resource-manager'
+    azure-resource-provider-folder: "./resource-manager"
```

Comment:
question? Do we need to have the "./" in the front of this name and the examples-directory? Or can it be removed to simplify these.

Reply:
I don't think so, the only place where this is used is to interpolate back into the `output-file` format

Reply:
Cool if we can remove those lets do it to make this simpiler.

Reply:
Updated the PR to remove `./`